tests/cypress/e2e/actions_users/registration_involved/case_28_review_pipeline_feature.js

```diff
@@ -291,6 +291,22 @@ context('Review pipeline feature', () => {
             cy.get('.cvat-issues-sidebar-previous-frame').should('be.visible').click();
             cy.checkFrameNum(1);
 
+            const framesWithIssues = [[0, 1, 'Demo 1'], [1, 1, customIssueDescription], [2, 1, customIssueDescription]];
+            for (const [frame] of framesWithIssues) {
+                cy.goCheckFrameNumber(frame);
+                if (frame === 1) {
+                    cy.collectIssueLabel().then((issueLabelList) => {
+                        for (let label = 0; label < issueLabelList.length; label++) {
+                            cy.resolveIssue(issueLabelList[label], 'Resolved issue');
+                        }
+                    });
+                }
+            }
```

Comment:
It seems the for loop is viable only for frame == 1, so we dont need to iterate all over the frames.
We can just:
```
cy.goCheckFrameNumber(1);
cy.collectIssueLabel().then((issueLabelList) => {
                        for (let label = 0; label < issueLabelList.length; label++) {
                            cy.resolveIssue(issueLabelList[label], 'Resolved issue');
                        }
                    });
```

Reply:
got it